qa/distros/all/ubuntu_20.04.yaml

```diff
@@ -1,2 +1,9 @@
 os_type: ubuntu
 os_version: "20.04"
+# the normal ubuntu 20.04 kernel (5.4.0-88-generic currently) have a bug that prevents the nvme_loop
+# from behaving.  I think it is this:
+#   https://lkml.org/lkml/2020/9/21/1456
+# (at least, that is the symptom: nvme nvme1: Connect command failed, error wo/DNR bit: 880)
+overrides:
+  kernel:
+    hwe: true
```

Comment:
@jdurgin that doesn't seem to work:

```
root@smithi063:~# lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal
root@smithi063:~# uname -a
Linux smithi063 5.4.0-94-generic #106-Ubuntu SMP Thu Jan 6 23:58:14 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
root@smithi063:~# apt-cache policy linux-generic-hwe-20.04
linux-generic-hwe-20.04:
  Installed: (none)
  Candidate: 5.13.0.25.26~20.04.12
```

Reply:
turns out i wasn't modifying the right file.

thanks @jdurgin @sebastian-philipp